include/language-deprecation.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Holds everything related to deprecated properties of `PLL_Language`.
+ *
+ * @since 3.4
+ */
+abstract class PLL_Language_Deprecation {
```

Comment:
I used `Deprecation` on purpose, because I usually use `Deprecated` for deprecated methods/properties. And this class contains only code that handles deprecation.
But on second thought, you're right, it's the same thing in the end, there is no reason to use another convention.
Done in [2a614b5](https://github.com/polylang/polylang/pull/1227/commits/2a614b5f9da4ae851fe2db707224483a6d580a57).